nerdlets/cloud-optimize-nerdlet/components/optimizationCandidates.js

```diff
@@ -1,211 +1,341 @@
-import React from 'react'
-import { Button, Table, Icon, Modal } from 'semantic-ui-react'
-import CsvDownload from 'react-json-to-csv'
-import _ from 'lodash'
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Table, Icon, Modal } from 'semantic-ui-react';
+import CsvDownload from 'react-json-to-csv';
+import _ from 'lodash';
 
-const monthlyHours = 720
+const monthlyHours = 720;
 
 export default class OptimizationCandidates extends React.Component {
+  static propTypes = {
+    instances: PropTypes.array,
+    header: PropTypes.string
```

Comment:
This one is stricky, it can be a string or an object, kind of funky.

This seems to work:
`header: PropTypes.oneOfType([PropTypes.string, PropTypes.object])`